mobile/README.md

```diff
@@ -23,6 +23,10 @@ This guide will help you to install a local build of Origin Wallet on a device o
 
 - Xcode Command Line Tools: `xcode-select --install`
 
+- Install CocoaPods: `sudo gem install cocoapods`
+
+- (!) For the project to load with cocoa pods dependancies be sure to open the `OriginCatcher.xcworkspace` file instead of `OriginCatcher.xcodeproj`
```

Comment:
Spelling, dependancies should be dependencies

Reply:
yup tnx 👍